global.json

```diff
@@ -1,11 +1,12 @@
 {
   "sdk": {
-    "version": "6.0.200",
+    "version": "7.0.100",
     "rollForward": "latestFeature"
   },
 
   "additionalSdks": [
     "3.1.416",
-    "5.0.405"
+    "5.0.405",
+    "6.0.200"
```

Comment:
It won't let me apply a suggestion because it thinks the current `5.0.405` line was deleted 😩 

We should probably build against the latest respective SDK versions:

- 3.1.425
- 5.0.408
- 6.0.403

Reply:
Actually need to take out the `5.0`, it's out of support now, so I'm proposing we no longer target/support it either.  Will up the remaining versions though.